scripts/pkg/.goreleaser.yml.template

```diff
@@ -1,22 +1,32 @@
 builds:
   - id: linux
-    binary: var/db/newrelic-infra/newrelic-integractions/bin/nri-{{.Env.NAME}}
+    binary: var/db/newrelic-infra/newrelic-integrations/bin/nri-{{.Env.NAME}}
     dir: nri-config-generator
     ldflags:
       - -X main.integration={{.Env.NAME}} -X main.integrationVersion=${.Tag} -X main.gitCommit={.Commit} -X main.BuildDate=${.Date}
     goos:
       - linux
-    # This field is automatically populated from the exporter.yaml
-    goarch: []
+    goarch: []  # This field is automatically populated from the exporter.yaml
+    hooks:
+      # This is ugly but needed.
+      # Templating folder path for AMD is easy with something like:
+      #   _{{ .Arch }}{{ with .Amd64 }}_{{ . }}{{ end }}
+      # But then it fails while trying to build ARM binaries with an error like this:
+      #   post hook failed: template: tmpl:4:66: executing "tmpl" at <.Amd64>: map has no entry for key "Amd64"
+      # So we had to copy it using a wildcard but inside a shell so the shell expand the wildcard.
+      post: bash -c "cp ./dist/linux_linux_{{ .Arch }}*/var/db/newrelic-infra/newrelic-integrations/bin/nri-{{.Env.NAME}} ./exporters/{{.Env.NAME}}/target/bin/linux_{{ .Arch }}"
```

Comment:
changing [dist folder](https://goreleaser.com/customization/dist/) would be an option?

Reply:
The dist field des not allow templating (we cannot use `{{ .Env.NAME}}` so we would need to "template it ourselves" as we do with the architecture. On top of that we would need to modify more stuff as the binary path is used both in goreleaser package job and in other scripts.